ggplot_recreate_challenge/01-import.R

```diff
@@ -0,0 +1,42 @@
+#=====================================================================#
+# This is code to create: 01-import.R
```

Comment:
The `01-import.R` file filters the data to the countries in the FT visualization. I did not cover this step in the `exercises.Rmd` file, but I can add it. I was concerned about having too much `dplyr` in this exercise, so I thought they should just load the filtered data (`FtExcessDeaths.csv`) and start visualizing.  

The `02-wrangle.R` and `03-visualize.R` files complete the same steps as the `solutions.Rmd`, but they need to be run sequentially (`02`, then `03`). I left these in there in case you wanted to show how a project like this might look outside of RMarkdown, but I can remove if you prefer.